plugins/com.google.cloud.tools.eclipse.appengine.libraries/plugin.xml

```diff
@@ -325,7 +325,24 @@
         <mavenCoordinates
               artifactId="google-cloud-translate"
               groupId="com.google.cloud"
-              version="1.3.1" />
+              version="LATEST" />
+      </libraryFile>
+    </library>
+
+    <library
+          id="cloudlogging"
+          group="clientapis"
+          name="%stackdriverlogging.name"
+          tooltip="%stackdriverlogging.tooltip"
+          javaVersion="1.8"
+          siteUri="https://cloud.google.com/translate/docs/reference/libraries" >
+      <libraryDependency id="googlecloudcore" />
```

Comment:
I don't think this one is so easy: [`com.google.cloud:google-cloud-logging:1.4.0`](https://repo1.maven.org/maven2/com/google/cloud/google-cloud-logging/1.4.0/google-cloud-logging-1.4.0.pom) shows that it requires:
   + `io.netty:netty-tcnative-boringssl-static:jar:2.0.3.Final`
   + `com.google.cloud:google-cloud-core-grpc:jar:1.4.0`
   + `com.google.api:gax-grpc:jar:0.22.0`
   + `com.google.api.grpc:proto-google-cloud-logging-v2:jar:0.1.15`
   + `io.grpc:grpc-netty:jar:1.4.0`
   + `io.grpc:grpc-stub:jar:1.4.0`
```